src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkService.java

```diff
@@ -22,6 +22,15 @@ public interface SnowflakeSinkService {
    */
   void startTask(String tableName, TopicPartition topicPartition);
 
+  /**
+   * Start a collection of TopicPartition. This should handle any configuration parsing and one-time
+   * setup of the task.
+   *
+   * @param partitions collection of topic partitions
+   * @param topic2Table a mapping from topic to table
+   */
+  void startTasks(Collection<TopicPartition> partitions, Map<String, String> topic2Table);
```

Comment:
I would not call this `startTasks`

SnowflaekSinkService is in instance associated to the task already. 

Lets call this `startPartitions()`

Reply:
Fixed.